jupyterhub/notebook-images/overlays/additional/params.env

```diff
@@ -0,0 +1,12 @@
+odh-minimal-notebook-image-v1=quay.io/modh/odh-minimal-notebook-container@sha256:a5a7738b09a204804e084a45f96360b568b0b9d85709c0ce6742d440ff917183
```

Comment:
To confirm, these are just fallback values, and won't require an update every release ..right?

Reply:
Yes, according to the plan, of updating the digest only on rhods-live-builder these variables on this repository we can consider as fallbacks.